records/board/96b_nitrogen.yaml

```diff
@@ -0,0 +1,11 @@
+common:
+    macros:
+        - IO_CONFIG_OVERRIDE
```

Comment:
What IO customization did you need to make for this board? Please comment in IO_Config_Override.h which pins changed from the default.

Reply:
Added "Nitrogen override" for the pins that changed from the default.